server/src/main/java/org/opensearch/cluster/coordination/FollowersChecker.java

```diff
@@ -308,6 +308,7 @@ private void handleDisconnectedNode(DiscoveryNode discoveryNode) {
         FollowerChecker followerChecker = followerCheckers.get(discoveryNode);
         if (followerChecker != null) {
             logger.info(() -> new ParameterizedMessage("{} disconnected", followerChecker));
+            clusterManagerMetrics.incrementCounter(clusterManagerMetrics.followerCheckAttemptFailureCounter, 1.0);
```

Comment:
This is not the followerCheck attempt here, it is taking an action to fail the node here.